code-of-conduct.md

```diff
@@ -1,43 +1,64 @@
-# OpenContainers Code of Conduct
+# OpenContainers Code of Conduct v1.0
```

Comment:
On Thu, Jun 09, 2016 at 11:31:27AM -0700, Chris Aniszczyk wrote:
> > @@ -1,43 +1,64 @@
> > -# OpenContainers Code of Conduct
> > +# OpenContainers Code of Conduct v1.0
> 
> I'd like to version things from now on so we can track it…

And the upstream version does:

  This Code of Conduct is adapted from the [Contributor
  Covenant][homepage], version 1.4…

> I think this is close enough to ship TBH :)

Please at least restore the reference definitions [1] (or look at the
rendering of your current version and decide that you really do want
the square brackets in the final paragraph [2]).

[1]: https://github.com/opencontainers/tob/pull/14#issuecomment-224960199
[2]: https://github.com/opencontainers/tob/blob/c1d92b78b102386430ebe407e4d14c9d87c407c2/code-of-conduct.md#thanks